packages/ckeditor5-ui/src/colorpicker/colorpickerview.ts

```diff
@@ -58,35 +61,38 @@ export default class ColorPickerView extends View {
 	declare public _hexColor: string;
 
 	/**
-	* Debounced event method. The `colorPickerEvent()` method is called the specified `waitingTime` after
-	* `debouncedPickerEvent()` is called, unless a new action happens in the meantime.
-	*/
```

Comment:
I tried to understand these docs but it appears they reference some non-existing methods or events. Could you please address this one?